bip-0178.mediawiki

```diff
@@ -3,7 +3,7 @@
   Layer: Applications
   Title: Version Extended WIF
   Author: Karl-Johan Alm <karljohan-alm@garage.co.jp>
-  Comments-Summary: No comments yet.
+  Comments-Summary: One comment, with some Discouragement
```

Comment:
Do you mean "Discouraged for implementation (one person)", or "Mixed review (one person)"?

Reply:
How do you say that it needs additional info and reasoning, otherwise in the current form of the specification I discourage it? At least it is not me who became discouraged :stuck_out_tongue_closed_eyes:

Reply:
"Discouraged for implementation (one person)"